src/Send/SendScreen/SendScreen.tsx

```diff
@@ -37,7 +37,7 @@ export const SendScreen = () => {
   const wallet = useSelectedWallet()
   const balances = useBalances(wallet)
 
-  const {utxos, isLoading, error} = useUtxos(wallet)
+  const utxos = useUtxos(wallet)
```

Comment:
now we have `wallet.utxos` sync, this will return a shallow copy of utxos, or we can subscribe to it using `useUtxos`